assets/components/legal/subscriptionTermsPrivacy/subscriptionTermsPrivacy.scss

```diff
@@ -0,0 +1,13 @@
+.component-subscription-terms-privacy__text {
+  color: gu-colour(neutral-4);
+  display: block;
+}
+
+.component-subscription-terms-privacy__href {
+  color: gu-colour(neutral-4);
```

Comment:
You can also use `color: inherit`in this one so it'll inherit it from the parent (which has the same text colour)